compatibility/bazel_tools/testing.bzl

```diff
@@ -546,6 +546,18 @@ excluded_test_tool_tests = [
             },
         ],
     },
+    {
+        # Self-service error code assertions adapted
+        "start": "1.17.1",
+        "platform_ranges": [
+            {
+                "end": "1.18.0-snapshot.20211102.8257.0.7391a3cd",
+                "exclusions": [
+                    "PackageManagementServiceIT",
+                ],
+            },
```

Comment:
Does this mean: "do not run PackageManagementServiceIT if ledger-api-test-tool is 1.17.1 or newer, against ledger server version 1.18.0-snapshot.20211102.8257.0.7391a3cd or older"?

Reply:
Updated it

Reply:
@pbatko-da how I understand it: does not run PackageManagementServiceIT if ledger-api-test-tool is older than 1.18.0-snapshot.20211102.8257.1 (inclusive) and if Ledger API is newer than the same version.